test/client/fixtures/sandbox/code-instrumentation/getters-test.js

```diff
@@ -134,36 +134,38 @@ test('document.referrer', function () {
 
 if (browserUtils.isWebKit) {
     test('url in stylesheet properties', function () {
-        var el       = document.createElement('div');
-        var url      = 'http://google.com/image.png';
-        var proxyUrl = urlUtils.getProxyUrl(url);
+        var el                 = document.createElement('div');
+        var url                = 'http://google.com/image.png';
+        var proxyUrl           = urlUtils.getProxyUrl(url);
+        var buildExpectedValue = function (value) {
+            var usedQuoteString = browserUtils.isSafari || browserUtils.isAndroid ? '' : '"';
```

Comment:
It's `quote`, not `userQuoteString`. And let's use feature detection to determine actual format of the quotes, so if it will change we will be notified about it.